### ISSUE TITLE
https://github.com/allinurl/goaccess is crashing

### ISSUE THREAD

**Issue by samuelstroschein** [jannesblobel/inlang-plugin-po#5]
_From https://github.com/inlang/inlang/issues/749#issue-1708944036_

> Message: "The reference language "en" must be included in the list of languages"

## Additional information 

The po plugin doesn't seem to include the reference language in the list of languages.

**Comment by samuelstroschein** [comment]
@jannesblobel we discussed it already but the po plugin needs to create "virtual" resources for the reference language. yes, annoying that those resources can be edited but as we both figured out `.po` is a bad format (little we can do about it)

**Comment by jannesblobel** [comment]
Yep, I'm going to change the plugin. It should also be ready tomorrow. But then we will have the problem with jsdelivr for a week: the version has to be updated [that takes a week](https://stackoverflow.com/questions/69148904/difference-in-jsdelivr-url-with-without-latest#:~:text=The%20caching%20period%20is%201%20week.).

**Comment by samuelstroschein** [comment]
@jannesblobel you can purge JSDelivr https://www.jsdelivr.com/

**Comment by jannesblobel** [comment]
Quick update: the problem is the file structure of this repo. The Reflanguage is "en", but the name of the reference file is 'goaccess.pot" instead of 'en'.  I am working on a good solution. 
<img width="907" alt="CleanShot 2023-05-18 at 19 58 52@2x" src="https://github.com/jannesblobel/inlang-plugin-po/assets/72493222/ce6396f6-c57a-4b09-90e7-c509cba7ff2b">
![CleanShot 2023-05-18 at 19 59 05@2x](https://github.com/jannesblobel/inlang-plugin-po/assets/72493222/37bea3cf-4b76-4440-b0e6-43f1278f7b15)
